dev_test_cmd.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+function outcome_and_exit {
```

Comment:
If you `set -e`, the script will exit on any failure. To print a status you can `trap "echo \$STATUS" EXIT`, and set status to `FAILED` by default, then change it to `PASSED` at the bottom of the script when everything is happy.